cpmonitor/migrations/0029_move_chart_images_and_resize_images.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 4.1.7 on 2023-09-09 13:17
+
+from django.db import migrations, models
+from pathlib import Path
+import os
+import shutil
+
+from PIL import Image
+
+
+def resize_images(apps, schema_editor):
+    upload_path = Path("cpmonitor") / "images" / "uploads"
```

Comment:
Would this work on the server? I would expect a variable to be used here.

Reply:
I was expecting migrations to be run with the Django root dir as working directory (as I would locally using python manage.py). If that is not guaranteed on the server then yes we would need a way to get the Django root dir. @fblampe do you know?

Reply:
The working directory is indeed the same, but the images are not in cpmonitor/images but in /images within the container. The MEDIA_ROOT property in config/settings/container.py can be used to refer to that location in the migration.